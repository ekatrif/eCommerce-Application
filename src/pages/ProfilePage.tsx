import React from 'react';
import Container from '../components/UI/container/Container';
import H1 from '../components/UI/titles/h1/H1';

const ProfilePage: React.FC = () => {
  return (
    <div className="content">
      <Container>
        <div className="main">
          <H1 text="Your Profile" />
        </div>
      </Container>
    </div>
  );
};

export default ProfilePage;
