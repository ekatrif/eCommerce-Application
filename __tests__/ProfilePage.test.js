import React from 'react';
import renderer from 'react-test-renderer';
import ProfilePage from '../src/pages/ProfilePage';

test('ProfilePage renders correctly', () => {
  const tree = renderer
    .create(<ProfilePage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
