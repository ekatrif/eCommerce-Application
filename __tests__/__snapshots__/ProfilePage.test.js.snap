// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfilePage renders correctly 1`] = `
<div
  className="content"
>
  <div
    className="container"
  >
    <div
      className="main"
    >
      <h1
        className="h1"
      >
        Your Profile
      </h1>
    </div>
  </div>
</div>
`;
